test-qwalk.py

```diff
@@ -262,9 +262,7 @@ def main() -> None:
         "Delete directory: %s/%s"
         % (parent_dir if parent_dir != "/" else "", test_dir_name)
     )
-    rc.fs.delete_tree(id_=test_dir["id"])
-    rc.fs.delete_tree(path=parent_dir + "/test-qwalk-copy")
-    rc.fs.delete_tree(path=parent_dir + "/copy-from-snap")
+    rc.tree_delete.create_job(test_dir["id"])
```

Comment:
Instead of just dropping support for the old API, could we check the version of the qfsd, and use the old or new API calls accordingly, so that customers who are running older build can still use this script?

Reply:
This is just for a test, so I think we're okay in terms of external compatibility. `qumulo_api~5.1` doesn't have `qumulo.rest_client.fs.delete_tree` either, so we'd have to check the module's version too somehow

Reply:
Oh oops, sorry I misread that, my bad!